### ISSUE TITLE
FoVBackgroundMaker used in tutorial without exclusion mask

### ISSUE THREAD

**Issue by luca-giunti** [gammapy/gammapy#2879]
**Gammapy version**
v.17

**Bug description**
Reminder issue: In the [analysis_1.ipynb](https://docs.gammapy.org/0.17/notebooks/analysis_1.html) the FoV background maker is used with no exclusion mask:

```
config.datasets.background.method = "fov_background"
config.datasets.background.parameters = {"method": "scale"}
```
We should either load or create one and apply it.

By the way, there is a typo either in cell [4] or in cell [18] : The correct latitude is 22.014 or 22.14?

**Comment by registerrier** [comment]
Since it is the first notebook, I would suggest not to make it more complex by creating the mask there. Instead, we should should add a mask in the gammapy-datasets and use it here.

**Comment by adonath** [comment]
@registerrier I think we should put a little script there to create it. The simplest thing might be just to download the HGPS significance maps and apply a hysteresis threshold?

**Comment by adonath** [comment]
Now that we have an exclusion mask here: https://github.com/gammapy/gammapy-data/blob/master/joint-crab/exclusion/exclusion_mask_crab.fits.gz the issue can be resolved quickly...

**Comment by adonath** [comment]
Fixed by #3380.
